pibooth/config/parser.py

```diff
@@ -283,21 +283,22 @@ def save(self, default=False):
                         val = self.get(section, name)
                     fp.write("# {}\n{} = {}\n\n".format(value[1], name, val))
 
-        self.enable_autostart(self.getboolean('GENERAL', 'autostart'))
+        self.enable_autostart()
 
     def edit(self):
         """Open a text editor to edit the configuration.
         """
         if open_text_editor(self.filename):
             # Reload config to check if autostart has changed
             self.read(self.filename, encoding="utf-8")
-            self.enable_autostart(self.getboolean('GENERAL', 'autostart'))
+            self.enable_autostart()
 
-    def enable_autostart(self, enable=True):
+    def enable_autostart(self):
```

Comment:
As it can no be used any more to enable/disable, name of the method should be changed: ``ensure_autostart`` or ``check_autostart``

Reply:
I changed the signature because it is called enable_autostart, for me "enable_something" will ... enable it

Reply:
But in this case, it can be enabled or disabled. It depends on the config

Reply:
I've put "handle_autostart", is it ok?